src/FacebookPixel/Assets.php

```diff
@@ -56,6 +58,17 @@ public static function loadFrontendAssets() {
 				GIVE_FBPT_VERSION,
 				true
 			);
+
+			$session    = new DonationAccessor();
+			$donation 	= new Donation( $session->getDonationId() );
```

Comment:
Please fix the formatting.

Reply:
Done, ref: https://github.com/impress-org/givewp-facebook-pixel-tracking/pull/10/commits/cccec9b3e85fc0a5b6d9397964b5a1cf801961a3